### ISSUE TITLE
Register with already registered email will send code 226

### ISSUE THREAD

**Issue by mjza** [isopropylcyanide/Jwt-Spring-Security-JPA#27]
When I try to register with an already used email it will send an error message with a success code 226

```
cause: "me.cimply.ask.odata.exception.ResourceAlreadyInUseException"
data: "Email already in use with Address : 'a1@gmail.com'"
success: false
timestamp: "2020-07-15T14:36:38.487Z"
```

**Comment by isopropylcyanide** [comment]
226 seems to be a good default. It doesn't communicate a successful intent like 200, 204 or 201

**Comment by mjza** [comment]
Any 2xx in java script will call the success callback function. In front-end normally we have fnError and fnSuccess. When you send 226 the success call back function will be executed.

**Comment by mjza** [comment]
For solving this issue I suggest to change the `@ResponseStatus(HttpStatus.IM_USED)` to `@ResponseStatus(HttpStatus.CONFLICT)` in both `AuthControllerAdvice.java` and `ResourceAlreadyInUseException.java` files. I didn't provided any pull request for it, if you think it is acceptable I can provide pull request.

**Comment by isopropylcyanide** [comment]
Yes I think a PR would make sense here. It is indeed a conflict that an email matches an existing one. Do you think you can raise one here?

**Comment by mjza** [comment]
OK. I will do.

**Comment by mjza** [comment]
Pull request is ready.

**Comment by isopropylcyanide** [comment]
Approved and merged.

**Comment by isopropylcyanide** [comment]
Merged #33
